libraries/chain/block_log.cpp

```diff
@@ -245,10 +246,13 @@ namespace eosio { namespace chain {
             //   block_id_type        previous;                   //bytes 14:45, low 4 bytes is big endian block number
             //   of previous block
 
-            EOS_ASSERT(position <= size(), block_log_exception, "Invalid block position ${position}",
-                       ("position", position));
+            file.seek_end(0);
```

Comment:
Why did you change this to not use `size()`, and do a `seek_end()` instead?

Reply:
It was a leftover from last move around. Reverted back.